apps/docs/pages/_app.tsx

```diff
@@ -66,6 +66,20 @@ function MyApp({ Component, pageProps }: AppPropsWithLayout) {
       <Favicons />
       {IS_PLATFORM || LOCAL_SUPABASE ? (
```

Comment:
not related to this PR but why do we have `LOCAL_SUPABASE` here? feels like its a dupe of `IS_PLATFORM`?
its only being used here in _app as well, unless LOCAL_SUPABASE !== IS_PLATFORM?

Reply:
there's no env var `NEXT_PUBLIC_LOCAL_SUPABASE` on vercel either - perhaps we can remove?

Reply:
Hmm yeah I wasn't too sure about LOCAL_SUPABASE either. @gregnr are you the person to ask here?

Reply:
no worries - lets just leave it here for now and revisit it later